doc/whats_new.rst

```diff
@@ -7,6 +7,22 @@ What's new?
 
 .. currentmodule:: hnn_core
 
+.. _0.2:
+
+0.2
+---
+
+Changelog
+~~~~~~~~~
+- Store all connectivity information under :attr:`~hnn_core.Network.connectivity` before building the network, by `Nick Tolley`_ in `#276 <https://github.com/jonescompneurolab/hnn-core/pull/276>`_
+
+Bug
+~~~
+
+API
+~~~
+- New API for defining cell-cell and drive-cell connections. Custom connections can be added with :func:`~hnn_core.Network.add_connection`, by `Nick Tolley`_ in `#276 <https://github.com/jonescompneurolab/hnn-core/pull/276>`_
```

Comment:
does it work also for drive-cell connections? I thought it was only for local connectivity ...

Reply:
Ah good catch, got my descriptions mixed up. Yeah `add_connection()` only accepts gids for the local network, but `net.connectivity` contains all connections, including drive-cell.